__tests__/campus-toggle.test.js

```diff
@@ -1,9 +1,12 @@
 import React from "react";
 import CampusToggle from "../src/components/campus-toggle/campus-toggle.component";
-import { CONCORDIA_RED, INACTIVE_BUTTON_COLOR } from "../src/constants/style";
+import {
+  CONCORDIA_RED,
+  INACTIVE_BUTTON_COLOR
+} from "../src/styles/colors.styles";
```

Comment:
This can be changed to `from "../src/styles/colors.styles";`

Reply:
Fixed

Reply:
Fixed